modules/client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/types/Project.scala

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.bluebrain.nexus.admin.client.types
+
+import ch.epfl.bluebrain.nexus.admin.client.types.Project._
+import ch.epfl.bluebrain.nexus.admin.ld.Const._
+import ch.epfl.bluebrain.nexus.admin.refined.ld.{AliasOrNamespace, Prefix}
+import ch.epfl.bluebrain.nexus.admin.refined.project.ProjectReference
+import io.circe.Decoder
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.refined._
+
+/**
+  * Data type exposing project information to external services.
+  *
+  * @param name           the name of the project
+  * @param prefixMappings the prefix mappings
+  * @param config         the configuration of the project
+  * @param rev            the selected revision for the project
+  * @param deprecated     the deprecation status of the project
+  */
+final case class Project(name: ProjectReference,
```

Comment:
I see the point of having a different `Project` here then the one defined in core, but I don't see the need of having a different `LoosePrefixMapping` and `Config`.

Reply:
the client should ideally just depend on the HTTP stack and not reuse any classes defined by the service (in this case core). I think @hygt should keep this additional set of classes.

Reply:
That was my idea. I now have removed the dependency on core in the build.sbt file.